config.toml.sample

```diff
@@ -1,22 +1,22 @@
 [app]
-    # Interface and port where the app will run its webserver.
-    address = "0.0.0.0:9000"
+# Interface and port where the app will run its webserver.
+address = "localhost:9000"
```

Comment:
Should explicitly document `0.0.0.0`. Something along the lines of, `To make listmonk accessible publicly over the internet, bind to 0.0.0.0:9000` or something. A vast majority of users who deploy are neither sysadmins nor developers, just hobbyists and this may not be apparent to them.

Reply:
True, it should be documented how to make the port accessible from the main IP, but I still think defaulting to `localhost` and documenting how they can switch to `0.0.0.0` for all interfaces or a specific IP would be better.